<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTemp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Température</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPres">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pression (hPa)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitre">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Projet Station Météo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mesures intérieures</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblExt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mesures extérieures</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtTemp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtPres">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblHum">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité (%)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtHum">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTempExt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Température</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtTempExt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPresExt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pression (hPa)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtPresExt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>140</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblHumExt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité (%)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtHumExt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTempMaxExt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp Max</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtTempMaxExt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTempMinExt">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp Min</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtTempMinExt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblVille">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ville</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDiffTemp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDiff">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Diff T int/ext</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDesc">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblClim">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>200</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 150, 247);</string>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cboChoixVille">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Paris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marseille</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lille</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Annecy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toulouse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Strasbourg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lyon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brest</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bordeaux</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ivry-sur-Seine</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblHeure">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Heure</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="cboPolice">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>370</y>
      <width>238</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>130</y>
      <width>173</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbCel">
       <property name="text">
        <string>Celcius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFah">
       <property name="text">
        <string>Fahrenheit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>180</y>
      <width>198</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbH12">
       <property name="text">
        <string>Heure 12h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbH24">
       <property name="text">
        <string>Heure 24h</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>230</y>
      <width>156</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="rbFra">
       <property name="text">
        <string>Français</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbEng">
       <property name="text">
        <string>English</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>115</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="rbJour">
       <property name="text">
        <string>Jour</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbNuit">
       <property name="text">
        <string>Nuit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblChoixVille">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix de la ville</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPara">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Paramètres</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblChoixPolice">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix de la police</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUnitTemp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblUnitTempExt">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblUnitTempMaxExt">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblUnitTempMinExt">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblUnitDiffTemp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>831</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
